apps/inv/src/assets/typography.scss

```diff
@@ -0,0 +1,75 @@
+// Mixin for typography Typography
```

Comment:
i think this file is not needed as we have tailwind plugins for typography tshering. It will be redundant if i am not wrong